test/navigation/NavigatorSpec.scala

```diff
@@ -901,6 +981,33 @@ class NavigatorSpec extends SpecBase with BeforeAndAfterEach {
 
             }
 
+            "if NIPHL is authorised and has NIPHL assessment and the answers are yes for all category 1 assessments" - {
+
+              val categoryInfoWithNiphlAssessments = CategorisationInfo(
+                "1234567890",
+                Seq(category1Niphl, category1),
+                Seq(category1Niphl, category1),
```

Comment:
Not valid scenario. Shouldn't ask Niphls question

Reply:
Updated